apps/frontend/src/app/5_pages/App/App.tsx

```diff
@@ -31,18 +42,15 @@ function App() {
           <div className="p-4">Hello.</div>
         </Dialog>
 
-        <Dropdown text="test">
-          <div>
-            <div className="my-2" onClick={noop}>
-              Dropdown Item 1
-            </div>
-            <div className="my-2" onClick={noop}>
-              Dropdown Item 2
-            </div>
-            <div className="my-2" onClick={noop}>
-              Dropdown Item 3
-            </div>
-          </div>
+        <Dropdown text={currentLang.toUpperCase()} className="my-4">
+          <Menu>
+            {languages.map(lng => (
+              <MenuItem
+                text={lng.toUpperCase()}
+                onClick={() => changeLanguage(lng)}
```

Comment:
```suggestion
                onClick={changeLanguage(lng)}
```

Reply:
will get the error `Type 'void' is not assignable to type 'MouseEventHandler<Element> | undefined'.ts` if I change this

Reply:
Even when `changeLanguage` is changed to?:  
```
const changeLanguage = useCallback((lng: string) => () => {
    i18next.changeLanguage(lng);
    setCurrentLang(lng);
  }, []);
```